lib/pager_api/pagination/kaminari.rb

```diff
@@ -66,7 +66,7 @@ def paginate_collection(collection, options = {})
           options[:page] = params[:page] || 1
           options[:per_page] = options.delete(:per_page) || params[:per_page] || ::Kaminari.config.default_per_page
 
-          collection.page(options[:page]).per(options[:per_page])
+          collection.send(PagerApi.page_access_method, options[:page]).send(PagerApi.per_page_access_method, options[:per_page])
```

Comment:
There is a `meta` which I think you can update to meet this two accessors,  what do you think?

Also there can you provide the update for the `will_paginate` driver module to meet this as well?

Thanks for the collaboration!